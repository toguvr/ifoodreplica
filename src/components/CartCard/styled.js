import styled from 'styled-components'

export const MainCartCardDiv = styled.div`
    width:100%;
    margin-top:8px;
    display:flex;
    flex-direction:column; 
    border-radius: 8px;
    border: solid 1px #b8b8b8;
    padding:16px;
`

export const CardTitle = styled.span`
    height: 18px;
    font-size: 16px;
    margin-bottom:9px;
    color: #e8222e;
`

export const CardDate = styled.span`
    height: 18px;
    font-size: 12px;
    color: #000000;
    margin-bottom:7px;
`

export const CardPrice = styled.span`
    height: 18px;
    font-size: 16px;
    color: #000000;
    font-weight:bold;
`