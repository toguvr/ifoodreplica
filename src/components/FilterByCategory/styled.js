import styled from 'styled-components'

export const HorizontalScroll = styled.div`
	display: flex;
	height: 42px;
	flex-wrap: nowrap;
  overflow-x: auto;
	margin-top: 8px;

	&::-webkit-scrollbar {
    display: none;
	}
	
	& button {
		flex: 0 0 auto;
	}

	&:last-child {
		margin-right: 5px;
	}
`
export const Button = styled.button`
	border: none;
	background-color: #fff;
	font-size: 16px;
	letter-spacing: -0.39px;
	margin: 0 16px;
	color: black;
	
	&:focus {
		color: ${props => props.color};
		outline: none;
	}

	
`
