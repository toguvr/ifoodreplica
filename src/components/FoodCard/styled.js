import styled from "styled-components"
import CardMedia from '@material-ui/core/CardMedia';


export const StyledCard = styled.div`
    width: calc(100% - 32px);
    min-height: 112px;
    display: flex;
    margin: 8px 16px 0;
    border: solid 1px #b8b8b8;
    border-radius:8px;
`

export const FlexDiv = styled.div`
    justify-content:space-between;
    display:flex;
`

export const StyledCardMedia = styled(CardMedia)`
    background-color: black;
    width: 30%;
    border-radius: 8px 0 0 8px;
`

export const CardContent = styled.div`
    width: 70%;
    display: flex;
    flex-direction:column;
    padding-left: 16px;
`

export const TypographyTitle = styled.span`
    margin-top: 18px;
    min-height:18px;
    font-size: 16px;
    color: #e8222e;
    white-space: nowrap;
    overflow: hidden;
    width: 167px;
    text-overflow: ellipsis;
`

export const RequestQuantityWrapper = styled.div`
    border: solid 1px #e8222e;
    display:flex;
    justify-content:center;
    align-items:center;
    width: 33px;
    height: 33px;
    border-radius: 0px 5px 0px 5px;
`

export const TypographyDescription = styled.span`
    margin: 8px 16px 0 0;
    display: -webkit-box;
    -webkit-line-clamp: 2;
    -webkit-box-orient: vertical;  
    overflow: hidden;
    height:32px;
    font-size: 12px;
    color: #b8b8b8;
`

export const TypographyValue = styled.span`
    margin-top: 4px;
    font-size: 16px;
    min-height: 19px;
    color: #000000;
`

export const TypographyButtonText = styled.span`
    font-size: 12px;
    min-height: 14px;
    color: ${props => props.borderColor};
`

export const AddRequestButton = styled.div`
    margin-top:7px;
    width:90px;
    height:31px;
    border-style:solid;
    border-width:thin;
    border-color: ${props => props.borderColor};
    border-radius: 8px 0px 8px 0px ;
    display:flex;
    justify-content:center;
    align-items:center;
`


