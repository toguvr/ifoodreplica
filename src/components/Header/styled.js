import styled from "styled-components";
import { ReactComponent as ArrowBack } from "../../assets/arrow-back.svg";
import ExitToAppIcon from '@material-ui/icons/ExitToApp';


export const Container = styled.div`
  height: 44px;
  box-shadow: 0 0.5px 0 0 rgba(0, 0, 0, 0.25);
  display: flex;
  align-items: center;
  justify-content: center;
  position:sticky;
  top:0;
  right:0;
  left:0;
  background: white;
`;

export const StyledArrow = styled(ArrowBack)`
	left: 0;
	margin-left: 16px;
  position: absolute;
`;

export const StyledExitToAppIcon = styled(ExitToAppIcon)`
	right: 0;
	margin-right: 16px;
  position: fixed;
`;

export const TitleWrapper = styled.div`
  align-self: center;
  width: 175px;
  height: 44px;
  display: flex;
  align-items: center;
  justify-content: center;

`;

export const Title = styled.p`
	font-size: 16px;
	letter-spacing: -0.39px;
  color: #000000;
`;
