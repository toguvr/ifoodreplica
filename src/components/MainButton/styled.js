import styled from 'styled-components'


export const ButtonDiv = styled.div`
    display:flex;
    justify-content:center;
    width:100%;
    
`

export const MainButton = styled.button`
    width: 100%;
    background: #e8222e;
    padding: 12px 16px;
    margin: 8px 16px 0;
    border:none;
    font-weight: normal;
    font-size: 16px;
    color: #000000;
    border-radius: 2px;
    :focus{
        outline:none;
    }
`