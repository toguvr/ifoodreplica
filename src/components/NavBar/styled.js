import styled from 'styled-components'

export const MainNavDiv = styled.div`
    display:flex;
    justify-content:space-around;
    align-items:center;
    width:100vw;
    border-top: solid 1px #b8b8b8;
    height:50px;
    position: fixed;
    bottom:0;
    background-color:white;
`
export const IconImg = styled.img`
    cursor:pointer;
`