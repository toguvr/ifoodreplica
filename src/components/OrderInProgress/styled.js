import styled from "styled-components";
import { ReactComponent as Clock } from '../../assets/clock.svg'

export const Wrapper = styled.div`
@import url('https://fonts.googleapis.com/css?family=Montserrat:400,700,800&display=swap');
  font-family: "Montserrat", sans-serif;
  position: fixed;
  bottom: 50px;
  display: flex;
  flex-direction: column;
  & p:not(:last-child) {
		margin-bottom: 8px;
  }
`;

export const Rectangle = styled.div`
  width: 100vw;
  height: 118px;
  background-color: #e8222e;
  display: flex;
  align-items: center;
  justify-content: flex-start;
`;

export const TextWrapper = styled.div`
	display: flex;
	flex-direction: column;
`

export const ContainerTeste = styled.div`
  width: 100vw;
  height: 49px;
	border-top: 1px solid black;
	padding: 10px;
`;

export const StatusOrder = styled.p`
  font-size: 16px;
  letter-spacing: -0.39px;
  color: #ffffff;
`;

export const RestaurantName = styled.p`
  font-size: 16px;
  letter-spacing: -0.39px;
  color: #000000
`;

export const SubTotal = styled.p`
	font-size: 16px;
	font-weight: 700;
	letter-spacing: -0.39px;
  color: #000000
`
export const StyledClock = styled(Clock)`
	color: #fff;
	margin: 24px;
`