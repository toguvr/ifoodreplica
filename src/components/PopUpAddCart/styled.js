import styled from 'styled-components'
import { SelectPicker } from "rsuite";

export const MainPopUpDiv = styled.div`
    min-height:100vh;
    width:100%;
    background: rgba(0,0,0,0.5);
    display:flex;
    flex-direction:column;
    align-items:center;
    justify-content:center;
    position: fixed;
    top:0;
    bottom:0;
`

export const SelectPickerStyled = styled(SelectPicker)`
    width: calc(100% - 32px);
    margin:0 16px;
`

export const WhiteContainer = styled.div`
    width:calc(100% - 32px);
    margin:0 16px;
    background-color: #ffffff;
    display:flex;
    flex-direction:column;
    align-items:center;
    position:relative;
`

export const Text = styled.span`
    height: 18px;
    font-size:16px;
    margin-top:43px;
    margin-bottom:31px;
`

export const SpanButton = styled.span`
    height: 19px;
    text-align: right;
    font-size:16px;
    margin-top:28px;
    width: calc(100% - 16px);
    color: #4f81a8;
    margin-right:16px;
    margin-bottom:21px;
    cursor: pointer;
`