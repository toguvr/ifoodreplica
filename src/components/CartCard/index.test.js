import React from 'react';
import renderer from 'react-test-renderer';
import { MainCartCardDiv, CardTitle, CardDate, CardPrice } from './styled';

describe("MainCartCardDiv", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<MainCartCardDiv />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("CardTitle", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<CardTitle />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("CardDate", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<CardDate />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("CardPrice", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<CardPrice />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})