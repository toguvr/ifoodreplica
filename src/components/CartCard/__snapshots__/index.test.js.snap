// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardDate Should match Snapshot 1`] = `
<span
  className="sc-htpNat iKXCly"
/>
`;

exports[`CardPrice Should match Snapshot 1`] = `
<span
  className="sc-bxivhb loNQxT"
/>
`;

exports[`CardTitle Should match Snapshot 1`] = `
<span
  className="sc-bwzfXH irRuWv"
/>
`;

exports[`MainCartCardDiv Should match Snapshot 1`] = `
<div
  className="sc-bdVaJa JtPMB"
/>
`;
